scripts/start-for-integration-tests.sh

```diff
@@ -31,11 +31,15 @@ echo "  docker logs fake-manage-recalls-api"
 echo "  docker logs redis"
 
 echo "Starting ${MANAGE_RECALLS_UI_NAME}, logs can be found here: ${LOG_FILE}"
-npm run start:feature >> "${LOG_FILE}" 2>&1 &
+npm run start:feature >>"${LOG_FILE}" 2>&1 &
```

Comment:
Do we need the `sleep` ? It worked without it

Reply:
Never did on my machine 😢